lib/kubeclient/common.rb

```diff
@@ -204,6 +204,16 @@ def retrieve_all_entities(entity_types)
         end
       end
 
+      def proxy_url(kind, name, port, namespace = '')
```

Comment:
@abonas yes in fact there was already this comment: https://github.com/abonas/kubeclient/pull/115#discussion_r38085297